src/app/explore/explore.component.html

```diff
@@ -23,6 +23,7 @@ <h1 class="b-rh-1 b-bold c-home-hero__heading">The leading online match funding
   <div class="b-container" *ngIf="campaigns">
     <app-filters
       [hasTerm]="hasTerm"
+      [isHomepage]="true"
```

Comment:
I've just realised this is saying that Explore is the home page. What I was anticipating when we discussed splitting out the pages and I split them on Jira was a new Home component which should be the only one working in `isHomepage` mode. So we have 3 grids:

* Home, which has a limited number and no filters and no infinite scroll, but directs you to Explore if you want to see more campaigns
* Explore which has filters and infinite scroll
* Meta-campaign which has filters and infinite scroll and is scoped to one parent campaign

Reply:
I've renamed the var to match it's intention so it's less confusing when we start creating a new home page component

Reply:
Ah, I think I was just confused about the order we're doing things @tbg-jasonfung. So what I think should continue to be Explore will have this set to `true` too. But for today, Explore continues to be the home page and so it's `false`. Let me know if that's wrong, as I think then we might still need to clarify?

Reply:
@NoelLH that's right, once we've split out the home component and explore component, then we can set that flag to `true` since we want clear filters button to show on the explore page.